File: CenaAlContrario/Models/Cena.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.Linq;
using System.Web;

namespace CenaAlContrario.Models
{
    public class Cena
    {
        [Key]
        public int CenaId { get; set; }
        public string Nome { get; set; }
        public DateTime DataInserimento { get; set; }
        public int Anno { get; set; }
        public int NumeroPartecipanti { get; set; }
        public string Localita { get; set; }
        public string Locale { get; set; }
        public virtual ICollection<Portata> Portatas { get; set; }
        public byte[] Foto { get; set; }
    }

    public class Portata
    {
        [Key]
        public int PortataId { get; set; }
        public Cena Cena { get; set; }
        public int Ordine { get; set; }
        public string Piatto { get; set; }
    }
}

File: CenaAlContrario/Scripts/Model/Cena.js
function Cena() {
}
function Cena(nome, localita, locale) {
    this.Nome = nome;
    this.Localita = localita;
    this.Locale = locale;
    this.Portatas = new Array();
}
Cena.prototype = {
    CenaId: "",
    Nome: "",
    DataInserimento: "",
    Anno: "",
    NumeroPartecipanti: "",
    Localita: "",
    Locale: "",
    Portatas: [],
    Foto: null,
    OwnedByUser: false
}

function Portata() { }
function Portata(ordine, piatto) {
    this.Ordine = ordine;
    this.Piatto = piatto;
}
Portata.prototype = {
    PortataId: "",
    Ordine: 0,
    Piatto: ""
}